packages/flame_lottie/example/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Miscellaneous
```

Comment:
There should be a .gitignore file on a global level now so that this isn't needed